packages/backend/src/modules/verifiers/VerifiersController.test.ts

```diff
@@ -47,6 +47,10 @@ const zksMock: ZkCatalogProject[] = [
           contractAddress: zkVerfifierAddress,
           chainId: ChainId.ETHEREUM,
           subVerifiers: [],
+          performedBy: {
+            name: 'L2BEAT',
+            link: 'https://l2beat.com',
+          },
```

Comment:
while this should be fine, it would be better to create a const with all verification performers
```tsx
const PerformedBy = {
	l2beat: { name: ..., link: ...}
	worldcoin: ... 
}
```
And in the end reference it:
```tsx
...
performedBy: PerformedBy.l2beat
...
```